src/index.js

```diff
@@ -219,18 +209,39 @@ ipcMain.handle(CLIPBOARD_CLEAR, clear);
 ipcMain.on(CLIPBOARD_EVENT, handleIPCCopy);
 ipcMain.on(CLIPBOARD_BULK_COPY, handleIPCBulk);
 
+/**
+ * @param {Object} config
+ * @param {string} config.title
+ * @param {Object} config.options
+ * @param {string} config.fileUrl
+ */
+const createPage = ({ title, options, fileUrl }) => {
```

Comment:
Rename `fileUrl` to `url`. It may be resolved to a file, or an internet URL. Just to be clear about the intended usage of it.